protopipe/scripts/write_dl1.py

```diff
@@ -278,8 +285,16 @@ class EventFeatures(tb.IsDescription):
                 feature_events[cam_id]["max_signal_cam"] = max_signals[tel_id]
                 feature_events[cam_id]["sum_signal_cam"] = moments.intensity
                 feature_events[cam_id]["N_LST"] = n_tels["LST_LST_LSTCam"]
-                feature_events[cam_id]["N_MST"] = n_tels["MST_MST_NectarCam"]
-                feature_events[cam_id]["N_SST"] = n_tels["SST"] # will change
+                feature_events[cam_id]["N_MST"] = (
+                    n_tels["MST_MST_NectarCam"]
+                    + n_tels["MST_MST_FlashCam"]
+                    + n_tels["SCT_SCT_SCTCam"]
```

Comment:
might be best to define a function like count_by_tel_class() that loops over and looks for "MST" rather than hard-coding this, but it seems there is a bug in ctapipe if SCT is called SCT_SCT_SCTCam... (it should be "MST_SCT_SCTCam").

Reply:
Done thanks!